campaign_themes/arctic.json

```diff
@@ -18,7 +18,8 @@
 						}
 					],
 					"default": "greenpeace"
-				}
+				},
+				"default": "minimal"
```

Comment:
I added a default to the configurations 🤔

Reply:
Be aware that these defaults currently need to be matched with the backend logic, as we still need to clean up handling of default values so that everything can be in one place in the config files.